Tasks/VsTest/distributedtest.ts

```diff
@@ -16,9 +17,23 @@ export class DistributedTest {
     }
 
     public runDistributedTest() {
+        this.publishCodeChangesIfRequired();
         this.registerAndConfigureAgent();
     }
 
+    private publishCodeChangesIfRequired(): number{
+        if (this.dtaTestConfig.tiaConfig.tiaEnabled) {
+            let testselector = new testselectorinvoker.TestSelectorInvoker();
+            let code = testselector.publishCodeChanges(this.dtaTestConfig.tiaConfig, null); //todo: enable custom engine
```

Comment:
if argument is not mandatory. mark it as optional so that you don't have to pass null